docs/source/tutorials/first_finetune_tutorial.rst

```diff
@@ -38,118 +44,120 @@ Then, it's as simple as:
 
 .. code-block:: bash
 
-  tune download \
-  meta-llama/Llama-2-7b \
-  --output-dir /tmp/llama2 \
-  --hf-token <ACCESS TOKEN>
+  tune download meta-llama/Llama-2-7b-hf \
+    --output-dir /tmp/Llama-2-7b-hf \
+    --hf-token <ACCESS TOKEN>
 
 This command will also download the model tokenizer and some other helpful files such as a Responsible Use guide.
 
-.. note::
-
-  You can also download the model directly through the Llama2 repository.
-  See `this page <https://llama.meta.com/get-started#getting-the-models>`_ for more details.
-
+|
 
 Selecting a recipe
 ------------------
 Recipes are the primary entry points for torchtune users.
-These can be thought of as end-to-end pipelines for training and optionally evaluating LLMs.
+These can be thought of as **singularly-focused scripts for interacting with LLMs**, including training
```

Comment:
```suggestion
These can be thought of as **singularly-focused scripts for interacting with LLMs**, including training,
```

Reply:
Do we want to put "hackable" or "extensible" or something like that as well? Basically just emphasize that they are intended to be experimented with